releasenotes/AzureDevOpsPipelineExtension.md

```diff
@@ -0,0 +1,14 @@
+## Azure App Configuration Extension release notes
```

Comment:
Can you change the folder name to `releaseNotes` from `releasenotes` ?

Reply:
It might make sense to put it in a folder named devops and make the file `AzureDevOpsTaskReleaseNotes`

Reply:
Basically, releaseNotes -> AzureDevOpsTaskReleaseNotes -> AzureDevOpsPipelineExtension.md? But I think the folder of `AzureDevOpsTaskReleaseNotes` is redundant. Also, we can use `releaseNotes` to groups all other projects' release notes, such as provider, cli. Make sense?

Reply:
Sounds good

Reply:
Great, I'll just change `releasenotes` to `releaseNotes` then.